README.md

```diff
@@ -1 +1,41 @@
-# dbt Formatter for pre-commit
\ No newline at end of file
+# dbt Formatter CLI
+
+## CLI for using dbt-formatter in pre-commit
+
+### Why?
+
+As we began writing (and reading!) more dbt code, we struggled with the problem of ensuring our models, macros, and snapshots were being written in a consistent way. One solution we found was[this dbt-formatter extension for vscode](https://marketplace.visualstudio.com/items?itemName=henriblancke.vscode-dbt-formatter&utm_source=VSCode.pro&utm_campaign=AhmadAwais). However, this was difficult to implement team-wide, as not everyone uses vscode and the autosave feature disables the autoformatting of files. Therefore, we decided to make a command line interface for the [dbt-formatter package](https://github.com/henriblancke/dbt-formatter). We've also added a Dockerfile and pre-commit-hooks.yaml for use with [pre-commit](https://pre-commit.com/).
+
+### Hot dog! So how do I use it?
+
+**As a CLI**
```

Comment:
this can be `####` so if github ever gets a useful table of contents macro, it will show these in the toc